src/client/lib/consentsTracking.ts

```diff
@@ -77,6 +77,7 @@ export const onboardingFormSubmitOphanTracking = (
   switch (`/${page}`) {
     case '/communication':
     case '/data':
+    case 'signin/success':
```

Comment:
missing the `/` at the start here i think

```suggestion
    case '/signin/success':
```

Reply:
Whoops, adjusted now